ts/parts-3d/SVGRenderer.ts

```diff
@@ -700,6 +710,23 @@ H.SVGRenderer.prototype.cuboidPath = function (
      * helper method to decide which side is visible
      * @private
      */
+    function mapSidePath(i: number): Highcharts.Position3dObject {
+        // Added support for 0 value in columns, where height is 0
+        //  but the shape is rendered
+        if (h === 0 && i > 1 && i < 6) {
+            return {
+                x: pArr[i].x,
+                y: pArr[i].y + 10,
```

Comment:
These hardcoded values: `6` and `10` - what those mean?

Reply:
I have commented the function in 78f4008